types/node/querystring.d.ts

```diff
@@ -19,6 +19,14 @@ declare module "querystring" {
 
     function stringify(obj?: ParsedUrlQueryInput, sep?: string, eq?: string, options?: StringifyOptions): string;
     function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): ParsedUrlQuery;
+    /**
+     * The querystring.encode() function is an alias for querystring.stringify().
+     */
+    function encode(obj?: ParsedUrlQueryInput, sep?: string, eq?: string, options?: StringifyOptions): string;
+    /**
+     * The querystring.encode() function is an alias for querystring.stringify().
```

Comment:
The comment is wrong here.

Reply:
Good call. Bad copy paste on my part :(. I'll approve your correction. Thanks for catching that!

Reply:
Sorry I got confused here. I'll push a correction then.

Reply:
Actually, no I'm confused where you're seeing this. I'm looking at my most recent commit and I don't see this:

https://github.com/methodbox/DefinitelyTyped/commit/e4f53b61d6ffffb8a2aadd86fd3e97ad30075cd1

Reply:
I look into the diff directly on GitHub. the file in v11 folder looks good but that one in node folder has the copy paste error.

Reply:
Okay, I’ve resolved it - seems like my browser was caching it or something :).